build.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh
+
+set -eu
+
+is_substring()
+{
+    case "$2" in
+        *$1*)
+        return 0
+        ;;
+        *)
+        return 1
+        ;;
+    esac
+}
+
+OK=0
+CURRENT_DIR=$(pwd)
+MY_NAME=$(basename -- "$0")
+ROOTFS_ARCHIVE="rootfs.tar.xz"
+BUILD_DIRECTORY="${CURRENT_DIR}/build"
+ROOTFS_DIRECTORY="${BUILD_DIRECTORY}/rootfs"
+DEBIAN_VERSION="jessie"
+
+cleanup()
+{
+    if is_substring "${CURRENT_DIR}" "${BUILD_DIRECTORY}"
+    then
+        rm -rf "${BUILD_DIRECTORY}"
+    fi
+}
+
+prepare()
+{
+    if [ ! -d "${ROOTFS_DIRECTORY}/usr/bin" ]
+    then
+        mkdir -p "${ROOTFS_DIRECTORY}/usr/bin"
+    fi
+
+    if [ ! -f "${ROOTFS_DIRECTORY}/usr/bin/qemu-arm-static" ]
+    then
+        cp /usr/bin/qemu-arm-static "${ROOTFS_DIRECTORY}/usr/bin/"
+    fi
+}
+
+
+bootstrap_rootfs()
+{
+    printf "Bootstrapping rootfs to %s\\n" "${ROOTFS_DIRECTORY}"
+    debootstrap \
+        --arch=armhf \
+        --variant=minbase \
+        --include=f2fs-tools,mtd-utils,busybox,udisks2,rsync \
+        "${DEBIAN_VERSION}" "${ROOTFS_DIRECTORY}" http://ftp.debian.com/debian
+}
+
+strip_rootfs()
+{
+    printf "Stripping rootfs\\n"
+    if [ -f "${ROOTFS_DIRECTORY}/usr/bin/qemu-arm-static" ]
+    then
+        printf "Removing qemu\\n"
+        # TODO: Strip even more see issue EMP-324 Reduce rootfs size
+        rm -f "${ROOTFS_DIRECTORY}/usr/bin/qemu-arm-static"
+    fi
+}
```

Comment:
No when bind mounting the file should exist and therefore it would not be more simple, it would make it more dangerous because something might happen to the original file.

Reply:
nah, read-only mount :)